foundry.toml

```diff
@@ -20,6 +20,7 @@
 
```

Comment:
*arbitrary comment location*

I know the whole README is still just the tempalte default. We need to update that. For now, can we at least include a section on how to run the script? Include where the network and deployer address are derived from, for example, and give an example for how to run it.

Reply:
Added the instructions to README here: [5b64a2d](https://github.com/showtime-xyz/creator-tokens/pull/40/commits/5b64a2d85b26c831690a7e67d65a0adcc2610c29)